File: classes/telepy.php
<?php

set_include_path(get_include_path().PATH_SEPARATOR.dirname(__FILE__).DIRECTORY_SEPARATOR.'libpy2php');
require_once 'libpy2php.php';
class telepy
{
    public function __construct()
    {
        try {
            require_once 'configparser.php';
        } catch (ImportError $e) {
            require_once 'ConfigParser.php';
        }
        require_once 'mtproto.php';
        $this->_config = $configparser->ConfigParser();
        if (!($this->_config->read('credentials'))) {
            pyjslib_printnl('File \'credentials\' seems to not exist.');
            $exit(-1);
        }
        $ip = $this->_config->get('App data', 'ip_address');
        $port = $this->_config->getint('App data', 'port');
        $this->_session = $mtproto->Session($ip, $port);
        $this->_session->create_auth_key();
        $__temp22 = py2php_kwargs_method_call($this->_session, 'method_call', ['get_future_salts'], ['num' => 3]);
        $this->_salt = $__temp22;
        $future_salts = $__temp22;
    }
}

File: original_telepy/classes/telepy.py

class Telepy():
  def __init__(self):
    # Deal with py2 and py3 differences
    try: # this only works in py2.7
      import configparser
    except ImportError:
      import ConfigParser as configparser
    import mtproto

    self._config = configparser.ConfigParser()
    # Check if credentials is correctly loaded (when it doesn't read anything it returns [])
    if not self._config.read('credentials'):
      print("File 'credentials' seems to not exist.")
      exit(-1)
    ip = self._config.get('App data', 'ip_address')
    port = self._config.getint('App data', 'port')

    self._session = mtproto.Session(ip, port)
    self._session.create_auth_key()

    self._salt = future_salts = self._session.method_call('get_future_salts', num=3)
